packages/tools/kadena-cli/src/devnet/utils/docker.ts

```diff
@@ -141,3 +141,13 @@ export function updateDevnet(version?: string): void {
   execSync(`docker pull ${image}`);
   console.log(chalk.green(`Updated ${image}`));
 }
+
+export function containerIsRunning(): boolean {
```

Comment:
When you use this method, are you making the assumption that the container is exposing a specific port as well? Because this method does not verify it, someone could have started a container using `kadena/devnet` but exposed it on a different port

Reply:
I am not using this method anymore. It was indeed not validating the port but the name before. Not it only validates if a container is running. If not useful, I can just remove the function because I am not using it

Reply:
Removed method since its not being used